arangod/RocksDBEngine/RocksDBIndex.h

```diff
@@ -90,6 +90,14 @@ class RocksDBIndex : public Index {
   Result remove(transaction::Methods* trx, LocalDocumentId const& documentId,
                 arangodb::velocypack::Slice const& doc, OperationMode mode) override {
     auto mthds = RocksDBTransactionState::toMethods(trx);
+#ifdef ARANGODB_ENABLE_FAILURE_TESTS
```

Comment:
You can safely remove the `#ifdef` here, because the macro `TRI_IF_FAILURE` contains that ifdef itself.